incubation.md

```diff
@@ -0,0 +1,110 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for
+inclusion in the main Jupyter organization. In this context a Subproject is a GitHub
+repository with a development team that follows the Jupyter governance and contribution model.
+The goals of the incubation process are:
+
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while
+  complying with corporate contribution restrictions.
```

Comment:
I wouldn't emphasize only corporate here, incubation applies in principle also to contribution from anybody. I really don't want to make it sound like this is only about corporate contributions.